templates/zerver/help/include/sidebar.md

```diff
@@ -76,15 +77,16 @@
 * [Change the stream description](/help/change-the-stream-description)
 * [Rename a stream](/help/rename-a-stream)
 * [Unsubscribe from a stream](/help/unsubscribe-from-a-stream)
-* [Change the privacy of a stream](/help/change-the-privacy-of-a-stream)
+* [Make a private stream public](/help/change-the-privacy-of-a-stream)
```

Comment:
This should possibly go to the anchor lower down on the page (i.e. directly go to relevant section of the doc).